langgraph/demo.py

```
@@ -0,0 +1,83 @@
+    except Exception as e:
+        print(f"An unexpected error occurred: {e}")
+
+    return results
+
+def repo_analyzer(repo_name: str):
+    """List out the team members from a certain team"""
+    return f"Successfully booked a stay at {repo_name}."
+
+def list_team_users(team_name: str):
+    """List out the team members from a certain specific team, for example Frontend Engineering"""
+    team_members_query_result = directory_db_query(f"SELECT users.name FROM users JOIN teams ON users.team_id = teams.id WHERE teams.name = \'{team_name}\'")
```

Ruff (0.12.2)

Possible SQL injection vector through string-based query construction

(S608)